docs/pipeline-api.md

```diff
@@ -12443,6 +12443,12 @@ string
 </tr>
 </tbody>
 </table>
+<h3 id="tekton.dev/v1beta1.TaskRunConditionType">TaskRunConditionType
+(<code>string</code> alias)</h3>
+<div>
+<p>TaskRunConditionType is an enum used to store TaskRun custom conditions
+conditions such as one used in spire results verification</p>
+</div>
```

Comment:
Is this due to, one of the previous commits missed codegen

Reply:
I think it might help to just delete this and run `./hack/update-codegen` again.